packages/core/src/sanitization/sanitization.ts

```diff
@@ -273,3 +274,45 @@ function getSanitizer(): Sanitizer | null {
   const lView = getLView();
   return lView && lView[ENVIRONMENT].sanitizer;
 }
+
+const attributeName: ReadonlyArray<string> = ['attributename'];
+
+/**
+ * @remarks Keep this in sync with DOM Security Schema.
+ * @see [SECURITY_SCHEMA](../../../compiler/src/schema/dom_security_schema.ts)
+ */
+const SECURITY_SENSITIVE_ELEMENTS: Readonly<Record<string, ReadonlyArray<string>>> = {
+  'iframe': ['sandbox', 'allow', 'allowfullscreen', 'referrerpolicy', 'csp', 'fetchpriority'],
+  'animate': attributeName,
+  'set': attributeName,
+  'animatemotion': attributeName,
+  'animatetransform': attributeName,
+};
+
+/**
+ * Validates that the attribute binding is safe to use.
+ *
+ * @param value The value of the attribute.
+ * @param tagName The name of the tag.
+ * @param attributeName The name of the attribute.
+ */
+export function ɵɵValidateAttribute(value: any, tagName: string, attributeName: string) {
```

Comment:
Lowercasing instruction name for consistency:
```suggestion
export function ɵɵvalidateAttribute(value: any, tagName: string, attributeName: string) {
```